app/views/movements.scala.html

```diff
@@ -38,9 +39,11 @@
   @if(notifications.isEmpty){
     @components.elements_in_separate_lines(submission.ucrBlocks.map(_.fullUcr))
   } else {
-    <a href="@routes.NotificationsController.listOfNotifications(submission.conversationId)">
-      @components.elements_in_separate_lines(submission.ucrBlocks.map(_.fullUcr))
-    </a>
+    @link(
+      message = components.elements_in_separate_lines(submission.ucrBlocks.map(_.fullUcr)),
+      textHidden = Some(messages("submissions.hidden.text", submission.ucrBlocks.map(_.fullUcr).mkString)),
```

Comment:
This is similar to the PR for Internal frontend. `mkString` without parameters will glue elements together.